docs/guides/subapps.md

```diff
@@ -0,0 +1,279 @@
+# Electrode Subapps
+
+Electrode Subapps is a mechanism for code splitting within a larger application. There are several advantages to creating a web application using subapps:
+
+- Superior code isolation is conducive to development across multiple teams
+- Smaller initial payloads loads through lazy-loadable subapps
+- Better application scalability
+
+## Getting Started
+
+To easily start using the npm initializer, run the following command:
+
+```bash
+mkdir my-app && cd my-app && npm init -y @xarc/app && npm i && npm run dev
+```
+
+This will create, install, and run an Electrode web application with a few simple subapps.
```

Comment:
Personally I like when this break down, highlights the command that was used above, so the engineer has context of how the above line was composed.

Reply:
I do like that! Would you separate into multiple lines and add comments to each line?